doc/EnvironmentVariables.md

```diff
@@ -79,7 +79,7 @@
 | RocksDB_MaxManifestFileSize               | Max size of a RocksDB MANIFEST file before it's rolled over                                                                                        | ulong                                                                 |                               |
 | RuntimeLogLevel                           | Runtime diagnostic logging level                                                                                                                   | fatal, error, warning, info, debug, verbose                           | info                          |
 | ShutdownWaitPeriod                        | Seconds to wait on shutdown before hard termination                                                                                                | int32                                                                 | 60                            |
-| SslProtocols                              | TLS protocol(s) to be supported                                                                                                                    | tls1.0, tls1.1, tls1.2, or any combination thereof separated by comma | all                           |
+| SslProtocols                              | TLS protocol(s) to be supported                                                                                                                    | tls1.2                                                                | all                           |
```

Comment:
"Values" column should return to what it was. "Default" column should be "tls1.2".